<launch>
    <param name="/use_sim_time" value="true" />
    <node pkg="walker_brain" type="grasp_cup_node" name="grasp_cup_node" output="screen">
    </node>

</launch>